<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_coffee">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>731</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="change_btn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>570</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить или добавить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
